src/theory/quantifiers/quant_module.h

```diff
@@ -40,7 +41,7 @@ class TermDb;
  * This is the virtual class for defining subsolvers of the quantifiers theory.
  * It has a similar interface to a Theory object.
  */
-class QuantifiersModule
+class QuantifiersModule : protected EnvObj
```

Comment:
Ditto, this doesn't use anything from `Env`?

Reply:
Ditto.